conda/recipes/dask-cuda/meta.yaml

```diff
@@ -24,11 +24,12 @@ requirements:
     - setuptools
   run:
     - python x.x
-    - dask-core >=2.4.0
+    - dask >=2.4.0
     - distributed >=2.18.0
     - pynvml >=8.0.3
     - numpy >=1.16.0
     - numba >=0.50.0,!=0.51.0
+    - pandas >=1.0,<1.2.0dev0
```

Comment:
Oh, perhaps you were referring to https://github.com/rapidsai/dask-cuda/pull/353/files#r488257470, @jakirkham . Yeah, I think this is now redundant and we can get rid of it.